### ISSUE TITLE
OSX: artefacts when selecting region.

### ISSUE THREAD

**Issue by lorenzgerber** [eclipse-swtchart/swtchart#418]
Example below from openchrom when selecting region for zoom.
![testest](https://github.com/user-attachments/assets/437a105d-1cc0-4548-b066-7ed56cb5b3f9)

**Comment by Mailaender** [comment]
What happens when you disable `Buffered Selection`?

**Comment by lorenzgerber** [comment]
Indeed, after switching off BufferedSelection in the Preferences and restarting the application, the above mentioned artefact does not occur anymore.

**Comment by Mailaender** [comment]
Can you please test again?

**Comment by lorenzgerber** [comment]
Does not occur anymore, tested on Intel OSX, Sequoia 15.7.9.
